controllers/controllers.articles.js

```diff
@@ -1,12 +1,23 @@
-const { selectArticles } = require('../models/models.articles.js');
+const { selectArticles, selectArticleById } = require('../models/models.articles.js');
 
 const getArticles = (req, res, next) => {
     selectArticles().then((articles) => {
         res.status(200).send({articles})
     })
+    .catch((err) => {
+        next(err)
+    })
+}
+const getArticleById = (req, res, next) => {
+    const article_id = req.params.article_id;
```

Comment:
You could destructure this value from params eg 
``` 
const { article_id } = req.params
``` 
Personal preference, I just wanted to show you a different approach

Reply:
done